app/flows/check_uk_visa_flow/outcomes/outcome_medical_y.erb

```diff
@@ -1,9 +1,21 @@
 <% text_for :title do %>
-  You’ll need a visa to visit the UK
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    You’ll need an electronic visa waiver (EVW) or a visitor visa
+  <% else %>
+    You’ll need a visa to visit the UK
+  <% end %>
 <% end %>
 
 <% govspeak_for :body do %>
-  You should apply for a [Standard Visitor visa](/standard-visitor).
+  <% if calculator.passport_country_is_saudi_arabia_or_bahrain? %>
+    You must either apply for:
+
+    + an [EVW](/get-electronic-visa-waiver) - you can apply from 1 June 2022
```

Comment:
Change [EVW] to [electronic visa waiver (EVW)] to make the link text more accessible.